lib/engine/game/base.rb

```diff
@@ -225,6 +225,8 @@ class Base
       # loans taken during ebuy can lead to receviership
       EBUY_CORP_LOANS_RECEIVERSHIP = false
 
+      SOLD_SHARES_DESTINATION = :bank
```

Comment:
Add a description and what choices are valid

Reply:
done